packages/twenty-front/src/modules/workflow/utils/getStepDefaultDefinition.ts

```diff
@@ -26,6 +26,27 @@ export const getStepDefaultDefinition = (
         },
       };
     }
+    case 'SEND_EMAIL': {
```

Comment:
packages/twenty-front/src/modules/workflow/utils/getStepDefaultDefinition.ts should be suffixed with `.util.ts`